README.md

```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+[Catherine Peppers] (https://github.com/catherinepeppers)
```

Comment:
Could you delete the space between your name and the URL? Looks great otherwise!

Reply:
Just did!